api/api.proto

```diff
@@ -1013,6 +1041,10 @@ message Volume {
   IoProfile derived_io_profile = 31;
   // InTrashcan if the volume is in trashcan
   bool in_trashcan = 32;
+  // vm migration support
+  Migration migration = 33;
+  // ExportedOn tracks attached volumes on multiple nodes for RWX and ROX volumes
+  repeated string exported_on = 34;
```

Comment:
Between attached_on and exported_on which field takes precedence?

Reply:
px is still single coordinator logic.
attached_on is primary active coordinator.
exported_on is all encompassing inclusive of passive coordinators.

most of the code and logic should rely on existing attached_on.
only those logic that needs to work for srb should look at exported_on